pkg/minishift/addon/addon_meta.go

```diff
@@ -89,6 +91,13 @@ func (meta *DefaultAddOnMeta) GetValue(key string) string {
 	return meta.headers[key].(string)
 }
 
+func (meta *DefaultAddOnMeta) OpenShiftVersion() string {
+	if val, contains := meta.headers[RequiredOpenShiftVersion].(string); contains {
+		return val
+	}
+	return ""
```

Comment:
So empty string is any version? Maybe make this a constant 'any_version = ""'?

Reply:
Should we allow empty sting to mean all? What if I just left it out by accident? Empty should maybe fail? Or ...

Default is all now?